### ISSUE TITLE
Not able to block youtube ads 

### ISSUE THREAD

**Issue by prorounak2005** [AdAway/AdAway#2839]
Adaway is not able to block youtube ads. I have to use Youtube Vanced for it. I am using vpn ad-blocker.

**Comment by zgfg** [comment]
Of course. But that is not a 'bug'

Please read and learn elswhere that YT uses different technology not pushing the ads from additional servers (ad-blockers like AdAway, DNS66, Blokada block those servers) but YT pushes the ads INSIDE the stream - hence they can only be blocked by a soecial application that works like a YT player, but removes the adds from the received stewam

**Comment by zgfg** [comment]
Btw, this is a pure duplicate of another 'bug' about the same (YT ads), just five items earlier than yours in the Open list

Not to mention that the same (YT ads) has already been reported, answered and closed in the past

Before reporting, research through the previous ones (open and closed) to avoid duplicates, non-issues, etc

**Comment by prorounak2005** [comment]
But it there a way to block youtube ads (without pihole). Because I have to use youtube vanced. I mean there is no difference between two but still I don't like it.

**Comment by zgfg** [comment]
There is no way to block YT sds by firewall, private DNS and ad-blockers that block particular sites

As wrote you above, it must be a modified YT player that will not show ads

I use Vanced root, if you don't like it google for other possible apps

And close this ticket

**Comment by prorounak2005** [comment]
ok thanks a lot for the help :)
